users/postgres/database.go

```diff
@@ -17,12 +17,17 @@ type database struct {
 	db *sqlx.DB
 }
 
+var (
+	errTransRollback = errors.New("failed to rollback transaction")
+)
+
 // Database provides a database interface
 type Database interface {
 	NamedExecContext(context.Context, string, interface{}) (sql.Result, error)
 	QueryRowxContext(context.Context, string, ...interface{}) *sqlx.Row
 	NamedQueryContext(context.Context, string, interface{}) (*sqlx.Rows, error)
 	GetContext(context.Context, interface{}, string, ...interface{}) error
+	BeginTxx(ctx context.Context, opts *sql.TxOptions) (*sqlx.Tx, error)
```

Comment:
Since you removed the code that uses this method, you can remove it from the interface and implementation, as well.